src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -66,7 +83,7 @@ public class TestSaga08 : Saga<TestSaga08.SagaData08>,
 
                 public Task Handle(StartSagaMessage message, IMessageHandlerContext context)
                 {
-                    return context.SendLocal(new SomeOtherMessage());
+                    return context.SendLocal(new StartSagaMessageByHandler());
```

Comment:
But there's no initiated saga upfront, is there? Do you mean that the first message should initiate the saga using the regular `IAmStartedByMessages` and the second should be used for correlation?